docs/specs/code/EXPORT_TRANSACTION_HISTORY_TO_FILE.md

```diff
@@ -14,12 +14,14 @@ I suggest adding a new "Export to File" icon at the top-right of the transaction
 
 When user clicks "Export to file" - new dialog window opens up:
 
-![image](https://user-images.githubusercontent.com/5585355/50386371-fb09fa00-06f5-11e9-970e-17c27da109c9.png)
+![image](https://user-images.githubusercontent.com/5585355/50401206-c6ea1400-079d-11e9-9980-2fb1c9cbc493.png)
+
 
 Dialog contains:
 1. Comment about the fact that the whole existing history will be exported
 2. Dropdown-selector: "File format"
 3. Button "Export"
+4. Icon "Help" (Question mark)
```

Comment:
good idea on adding a help button

Reply:
I didn’t see what it should contain. Maybe we could move it to the second iteration? so we have time to give it a better thought before EOY

Reply:
Was added just as a response to the idea that export format might not be obvious to users by default. But moving it to i2 sounds reasonable 👌